sei-db/ss/pebbledb/db.go

```diff
@@ -77,6 +80,22 @@ type Database struct {
 	lastRangeHashedCache int64
 	lastRangeHashedMu    sync.RWMutex
 	hashComputationMu    sync.Mutex
+
+	// Metrics collection
+	metricsCtx        context.Context
+	metricsCancel     context.CancelFunc
+	lastMetricsValues metricsSnapshot
+	metricsMu         sync.RWMutex
```

Comment:
This is a read/write mutest but is only used for write. For write, it is only used to control access to `lastMetricsValues` which is a field we do not need to store as a struct field? see comment on `collectAndRecordMetrics`.